client/src/partials/patients/record/patient_record.js

```diff
@@ -1,26 +1,59 @@
 angular.module('bhima.controllers')
 .controller('PatientRecordController', PatientRecordController);
 
-PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment'];
+PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment', 'Upload', '$timeout', '$translate'];
 
-function PatientRecordController($stateParams, Patients, Notify, moment) {
+function PatientRecordController($stateParams, Patients, Notify, moment, Upload, $timeout, $translate) {
   var vm = this;
   var patientID = $stateParams.patientID;
   
   vm.loading = true;
+  vm.uploadFiles = uploadFiles;
+  vm.textButton = $translate.instant('PATIENT_RECORDS.UPLOAD_PICTURE');
+ 
+  refreshPatientsRead();
 
-  /** @fixme if no uuid is provided this will download all the patients through the base url '/' */
-  Patients.read(patientID)
-    .then(function (result) {
-      vm.patient = result;
-      vm.loading = false;
+  function uploadFiles(file, errFiles) {
+    vm.f = file; 
```

Comment:
Whilst I'm looking at this function - I don't think this is used after being initialised.